core/widgetdiv.js

```diff
@@ -16,201 +16,232 @@
  * @name Blockly.WidgetDiv
  * @namespace
  */
-goog.provide('Blockly.WidgetDiv');
+goog.module('Blockly.WidgetDiv');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.common');
-goog.require('Blockly.utils.dom');
-
-goog.requireType('Blockly.utils.Rect');
-goog.requireType('Blockly.utils.Size');
-goog.requireType('Blockly.WorkspaceSvg');
+/* eslint-disable-next-line no-unused-vars */
+const Rect = goog.requireType('Blockly.utils.Rect');
+/* eslint-disable-next-line no-unused-vars */
+const Size = goog.requireType('Blockly.utils.Size');
+/* eslint-disable-next-line no-unused-vars */
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const common = goog.require('Blockly.common');
+const deprecation = goog.require('Blockly.utils.deprecation');
+const dom = goog.require('Blockly.utils.dom');
 
 
 /**
  * The object currently using this container.
  * @type {Object}
- * @private
  */
-Blockly.WidgetDiv.owner_ = null;
+let owner = null;
 
 /**
  * Optional cleanup function set by whichever object uses the widget.
  * @type {Function}
- * @private
  */
-Blockly.WidgetDiv.dispose_ = null;
+let dispose = null;
 
 /**
  * A class name representing the current owner's workspace renderer.
  * @type {string}
- * @private
  */
-Blockly.WidgetDiv.rendererClassName_ = '';
+let rendererClassName = '';
 
 /**
  * A class name representing the current owner's workspace theme.
  * @type {string}
- * @private
  */
-Blockly.WidgetDiv.themeClassName_ = '';
+let themeClassName = '';
+
+/**
+ * The HTML container for popup overlays (e.g. editor widgets).
+ * @type {!Element}
+ */
+let DIV;
+/** @deprecated September 2021 */
+exports.DIV = DIV;
+
+const getDiv = function() {
+  return DIV;
+};
+exports.getDiv = getDiv;
+
+const setDiv = function(newDiv) {
+  DIV = newDiv;
+};
```

Comment:
This seems like a place where we could make a decision make a setter only for tests.
I don't think the original intention was to allow this property to be reassigned. It is set by the `createDom` method that is called in `inject`.
However, I'm not sure whether anyone is currently reassigning this property.

Reply:
Yeah, I agree that seems better - updated accordingly.